kubeyard/minikube.py

```diff
@@ -92,31 +92,24 @@ def _start_env_as_arguments(self):
         }
         return ['{}={}'.format(key, value) for key, value in env.items()]
 
-    @staticmethod
-    def _apply_kube_dns_fix():
-        """
-        Fix needed on Ubuntu with systemd-resolved.
-        https://github.com/kubernetes/minikube/issues/2027
-        """
-        logger.info('Applying kube-dns fix...')
-        fix_path = pathlib.Path(__file__).parent / 'definitions' / 'kube-dns-fix'
-        sh.kubectl('apply', '--record', '-f', fix_path)
-        logger.info('kube-dns fix applied')
-
-    def _apply_dashboard_fix_if_needed(self):
-        """
-        Fix needed on minikube 0.25.
-        https://github.com/kubernetes/dashboard/issues/2767
-        """
-        logger.info('Applying dashboard fix...')
+    @cached_property
+    def _id(self):
+        return str(sh.id())
+
+    @cached_property
+    def uid(self) -> str:
```

Comment:
I suppose we can do Python-native with `os.getuid()` and `os.getguid()`.

Reply:
Ok. I'll change all other usages in separate commit.